pandas/core/frame.py

```diff
@@ -2719,9 +2730,13 @@ def assign(self, **kwargs):
         or modified columns. All items are computed first, and then assigned
         in alphabetical order.
 
-        .. versionchanged :: 0.23.0
+        .. versionchanged :: 0.23.0.
 
-            Keyword argument order is maintained for Python 3.6 and later.
+        Keyword argument order is maintained for Python 3.6 and later.
+
+        See Also
+        --------
+        DataFrame.assign: For column(s)-on-DataFrame operations
```

Comment:
This is the docstring of DataFrame.assign I think? So then not needed to refer to itself